doc/development/rfc/version_numbering.md

```diff
@@ -0,0 +1,184 @@
+# RFC: Version Numbering
+
+Author: Vaclav Petras
+
+Status: Draft
+
+## Summary
+
+Use version number in a format _major.minor.micro_ where
+- _major_ is for large additions or backwards incompatible changes,
+- _minor_ is for additions, and
+- _micro_ is for fixes.
```

Comment:
I wonder if we 'd better follow SemVer with "patch" rather than "micro"?

Reply:
I'm open to it.

I think the idea for micro was to emphasize that minor is not that small. Patch also has a different meaning like diff or commit although that seems to be diminishing given the spread of PRs/MRs. In the old days, we also had plenty of new features in the "patch" releases than just patches, so micro just stays neutral in that regard.

Reply:
Keeping micro for now.